addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -13,16 +13,21 @@ class AccountMoveReversal(models.TransientModel):
     l10n_latam_document_number = fields.Char(string='Document Number')
     l10n_latam_manual_document_number = fields.Boolean(compute='_compute_l10n_latam_manual_document_number', string='Manual Number')
 
-    @api.depends('l10n_latam_document_type_id')
+    @api.depends('l10n_latam_document_type_id', 'journal_id')
     def _compute_l10n_latam_manual_document_number(self):
         self.l10n_latam_manual_document_number = False
-        for rec in self.filtered('move_ids'):
-            move = rec.move_ids[0]
-            if move.journal_id and move.journal_id.l10n_latam_use_documents:
-                rec.l10n_latam_manual_document_number = move._is_manual_document_number()
+        for wiz in self.filtered(lambda x: x.journal_id and x.journal_id.l10n_latam_use_documents):
+            wiz.l10n_latam_manual_document_number = self.env['account.move'].new({
+                'move_type': wiz._reverse_type_map(wiz.move_ids[0].move_type),
+                'journal_id': wiz.journal_id[0].id,
+                'partner_id': wiz.move_ids.partner_id[0].id,
+                'company_id': wiz.move_ids.company_id[0].id,
```

Comment:
```suggestion
                'journal_id': wiz.journal_id.id,
                'partner_id': wiz.move_ids[0].partner_id.id,
                'company_id': wiz.move_ids[0].company_id.id,
```
although it is very much an edge case, I'd suggest taking the partner of the first move rather than the first partner of all the moves, as there might be no partner set on any of the moves in which case this will traceback.

Reply:
Thank you again for your suggestion! Sorry I missed that case. I've already pushed the changes!